lib/data/wrapper.js

```diff
@@ -158,14 +158,15 @@ const data = {
         client.healthcheck(log, (err, result) => {
             const respBody = {};
             if (err) {
-                log.error(`error from ${implName}`, { error: err });
-                respBody[implName] = {
+                log.error('error from sproxyd', { error: err });
```

Comment:
Because 'implName' can be 'mem' or 'file', which wasn't descriptive enough in the response. If the other backends are running in mem as well, then you wouldn't be able to tell which response came from which backend.